docs/kpm_oci.md

```diff
@@ -0,0 +1,76 @@
+# Use OCI-based Registries
+
+Beginning in kpm v0.2.0, you can use container registries with OCI support to store and share kcl packages.
```

Comment:
Here it is mentioned that the version of kpm is v0.2.0, but the version displayed in gif is v0.0.1.